utils/check/handle_rpms.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+set -uex
+
+# This file does an install of the RPMs found in "${REPO_DIR}" and then
+# finds all the packages that the binaries installed reference.
+
+# The file ${PACKAGES} will contain a list of packages that the RPMs depend
+# on.
+
+# The file ${SOFILES} will contain a list of sharable images depended on by
+# the RPMs.  The default name for ${SOFILES} is "daos_depends_libries".
+
+# This will likely need to be edited for other build systems.
+
+: "${WORKSPACE:="${PWD}"}"
+
+: "${RPM_DIR:="${WORKSPACE}/rpm_dir"}"
+
+: "${SOFILES:="daos_depends_libraries"}"
+: "${PACKAGES:="daos_depends_packages"}"
+
+
+repo_file="/etc/yum.repos.d/"
+repo_file+="repo.dc.hpdd.intel.com_repository_daos-stack-"
+repo_file+="el-7-x86_64-stable-local.repo"
+
+repo_url="https://repo.dc.hpdd.intel.com/repository/"
+repo_url+="daos-stack-el-7-x86_64-stable-local"
+
+if [ ! -e "${repo_file}" ]; then
+  cat << EOF > daos_repo_file
+
+[repo.dc.hpdd.intel.com_repository_daos-stack-el-7-x86_64-stable-local]
+name=added from: ${repo_url}
+baseurl=${repo_url}
+enabled=1
+
+gpgcheck = False
+
+EOF
+  sudo cp daos_repo_file "${repo_file}"
+  sudo yum -y makecache
```

Comment:
Shouldn't need this.  The first operation to use the repo does it implicitly.

Reply:
Changed.